### PR TITLE
🐛 FIX: Should sync package when registry id is null

### PR DESCRIPTION
**Pull request by fengmk2**

### CODE REVIEW

**Comment by fengmk2** [comment]
content:2022-10-06 10:32:24,329 WARN 489021 [-/127.0.0.1/b56451b0-450f-11ed-813f-7d3d65304315/6615485.653ms SCHEDULE /__schedule?path=/home/admin/application/app/port/schedule/ChangesStreamWorker.js&interval=60000&type=worker&env=&disable=false&immediate=false] [ChangesStreamService.executeSync:skip] package binary-mirror-config is not in registry 63064513c65c21ab03a7324a

File: app/core/service/PackageSyncerService.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ export class PackageSyncerService extends AbstractService {
     const [ scope, name ] = getScopeAndName(fullname);
     const pkg = await this.packageRepository.findPackage(scope, name);
     // sync task request registry is not same as package registry
-    if (pkg && options?.registryId) {
+    if (pkg && pkg.registryId && options?.registryId) {
```
Comment (fengmk2) [inline app/core/service/PackageSyncerService.ts:73]: pkg.registryId maybe null

**Comment by fengmk2** [comment]
egg-mock 的 mockHttpclient 有兼容性问题，我修复一下。

**Review by killagu — APPROVED** [review]
LGTM

**Comment by fengmk2** [comment]
2.3.1

**Pull request merged**